content/en/docs/tutorials/security/seccomp.md

```diff
@@ -196,13 +197,20 @@ If you were introducing this feature into production-like cluster, the Kubernete
 recommends that you enable this feature gate on a subset of your nodes and then
 test workload execution before rolling the change out cluster-wide.
 
-More detailed information about a possible upgrade and downgrade strategy can be
-found in the [related Kubernetes Enhancement Proposal (KEP)](https://github.com/kubernetes/enhancements/tree/a70cc18/keps/sig-node/2413-seccomp-by-default#upgrade--downgrade-strategy).
+You can find more detailed information about a possible upgrade and downgrade strategy
+in the related Kubernetes Enhancement Proposal (KEP):
+[Enable seccomp by default](https://github.com/kubernetes/enhancements/tree/9a124fd29d1f9ddf2ff455c49a630e3181992c25/keps/sig-node/2413-seccomp-by-default#upgrade--downgrade-strategy).
 
-Since the feature is in alpha state it is disabled per default. To enable it,
-pass the flags `--feature-gates=SeccompDefault=true --seccomp-default` to the
-`kubelet` CLI or enable it via the [kubelet configuration
-file](/docs/tasks/administer-cluster/kubelet-config-file/). To enable the
+Seccomp defaulting for Pods is a beta feature in Kubernetes {{< skew currentVersion >}}, 
```

Comment:
Question: Is the feature called 'Seccomp defaulting' or 'Seccomp'?
I'm not sure I understand the use of the word, 'defaulting'. Which or what is 'defaulting'?
`However, you still need to enable this defaulting for each node where
you would like to use it.`

Reply:
Right, fixing it in https://github.com/kubernetes/website/pull/35121